Samples/iOS-Swift/iOS-Swift/AppDelegate.swift

```diff
@@ -55,7 +55,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             options.sessionTrackingIntervalMillis = 5_000
             options.attachScreenshot = true
             options.attachViewHierarchy = true
+            
 #if targetEnvironment(simulator)
+            options.enableSpotlight = true
```

Comment:
As the default setup sends to localhost, I think it currently only makes sense to enable Spotlight when running on a simulator because otherwise, it will also try to send to localhost for the UI tests. We can change this anytime.